inferno-core/src/Inferno/Parse.hs

```diff
@@ -362,6 +362,36 @@ array p = label "array\nfor example: [1,2,3,4,5]" $
           )
         <|> pure []
 
+record :: Parser a -> Parser (SourcePos, [(Ident, a, Maybe SourcePos)], SourcePos)
+record p = label "record\nfor example: {name: \"Zaphod\", age: 391}" $
```

Comment:
Still haven't changed the parser to parse records as `{x = 1.1, y = 2.2}` and record field access as `rec.field`